frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1465,9 +1735,10 @@ function ReaderFooter:addToMainMenu(menu_items)
                     separator = separator ~= "" and separator or "none"
                     return T(_("Item separator: %1"), separator)
```

Comment:
```suggestion
                    return T(_("Item delimiter: %1"), separator)
```
?

Reply:
Isn't a delimiter like parentheses or quotation marks? As in, here's the start and there's the finish. If it sounds better I suppose the distinction doesn't really matter.

Reply:
> Isn't a delimiter like parentheses or quotation marks?

"It's a subset of them"™, but commas, colons can also be delimiters for example.

Reply:
What I mean is conceptually, not any specific character

bla | lala

is a separator

vs

'bla' 'lala'

with delimiters

Reply:
So yay or no-yay?

Reply:
cool.